core-services/openshift-monitoring/cfg_prometheus-prow-rules_prometheusrule.yaml

```diff
@@ -72,3 +72,33 @@ items:
           sum(rate(apiserver_request_latencies_summary_sum{job="apiserver",subresource!="log",verb!~"^(?:LIST|WATCH|WATCHLIST|PROXY|CONNECT)$"}[5m])) without(instance, pod)
           /
           sum(rate(apiserver_request_latencies_summary_count{job="apiserver",subresource!="log",verb!~"^(?:LIST|WATCH|WATCHLIST|PROXY|CONNECT)$"}[5m])) without(instance, pod)
+    - name: release-controller-down
+      rules:
+      - alert: releaseControllerDown
+        expr: |
+          kube_deployment_status_replicas_unavailable{namespace="ci", deployment="release-controller"} == 1
```

Comment:
Can we make this >= 1?

Reply:
And/or a glob on the deployment name? Then in the alert text use the deployment name to alern on which?